### ISSUE TITLE
MySQL: Table exclusions in `EXTRA_OPTS` causes infinite retry loop

### ISSUE THREAD

**Issue by dabico** [tiredofit/docker-db-backup#234]
### Summary

I want to exclude some tables from my export by adding the `--ignore-table` flags to `EXTRA_OPTS`.
What ends up happening is the process enters an infinite retry loop. I saw that there was a similar issue (#208) in the past.
Has this been fixed?

### Steps to reproduce

Add `EXTRA_OPTS: "--ignore-table=db_name.first_table_name --ignore-table=db_name.second_table_name"` to the docker-compose `environment` (or docker command equivalent).

### What is the expected *correct* behavior?

Service should produce DB dump without the specified tables, while also not entering a retry loop.

### Relevant logs and/or screenshots

<details>
<summary>Error Log</summary>

```log
2023-06-26T09:42:56.144771966Z                                                                        ,---.
2023-06-26T09:42:56.144800684Z ,--------.,--.                 ,--.            ,---.    ,--. ,--------.|   |
2023-06-26T09:42:56.144805020Z '--.  .--'`--',--.--. ,---.  ,-|  |     ,---. /  .-'    |  | '--.  .--'|  .'
2023-06-26T09:42:56.144808074Z    |  |   ,--.|  .--'| .-. :' .-. |    | .-. ||  `-,    |  |    |  |   |  |
2023-06-26T09:42:56.144810597Z    |  |   |  ||  |   \   --.\ `-' |    ' '-' '|  .-'    |  |.--.|  |   `--'
2023-06-26T09:42:56.144813203Z    `--'   `--'`--'    `----' `---'      `---' `--'      `--''--'`--'   .--.
2023-06-26T09:42:56.144815537Z                                                                        '--'
2023-06-26T09:42:56.144852666Z Image:  tiredofit/db-backup | Version  3.9.6 Type 'image_changelog' for details
2023-06-26T09:42:56.144865481Z Repository/Documentation: https://github.com/tiredofit/docker-db-backup/
2023-06-26T09:42:56.144869145Z 
2023-06-26T09:42:56.146294045Z If this image provides you value  - Consider sponsoring my work for continued 
2023-06-26T09:42:56.146328103Z development, timely updates, and feature requests. Commercial support available.
2023-06-26T09:42:56.146335335Z 
2023-06-26T09:42:56.146338060Z                     More Info:  https://www.tiredofit.ca
2023-06-26T09:42:56.146348880Z 
2023-06-26T09:42:56.216291216Z 2023-06-26.09:42:56 [NOTICE] ** [timezone] Timezone: Setting to 'Etc/UTC' from 'Etc/GMT'
2023-06-26T09:42:56.413222825Z 2023-06-26.09:42:56 [NOTICE] ** [scheduling] Container configured for scheduled tasks with 'cron'
2023-06-26T09:42:57.105352244Z 2023-06-26.09:42:57 [STARTING] ** [scheduling] [1] Starting cron
2023-06-26T09:43:01.992111249Z 2023-06-26.09:43:01 [INFO] ** [db-backup] Next Backup at 2023-06-26 09:43:01 UTC
2023-06-26T09:43:07.017400424Z 2023-06-26.09:43:07 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (5 seconds so far)
2023-06-26T09:43:12.032868423Z 2023-06-26.09:43:12 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (10 seconds so far)
2023-06-26T09:43:17.050603228Z 2023-06-26.09:43:17 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (15 seconds so far)
2023-06-26T09:43:22.067703559Z 2023-06-26.09:43:22 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (20 seconds so far)
2023-06-26T09:43:27.081039724Z 2023-06-26.09:43:27 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (25 seconds so far)
2023-06-26T09:43:32.092125051Z 2023-06-26.09:43:32 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (30 seconds so far)
2023-06-26T09:43:37.105125444Z 2023-06-26.09:43:37 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (35 seconds so far)
2023-06-26T09:43:42.115842809Z 2023-06-26.09:43:42 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (40 seconds so far)
2023-06-26T09:43:47.130181781Z 2023-06-26.09:43:47 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (45 seconds so far)
2023-06-26T09:43:52.151928244Z 2023-06-26.09:43:52 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (50 seconds so far)
2023-06-26T09:43:57.167211538Z 2023-06-26.09:43:57 [WARN] ** [db-backup] MySQL/MariaDB Server 'gse-db' is not accessible, retrying.. (55 seconds so far)
...
```

</details>

### Environment

Images:
- mysql:8.0.33
- tiredofit/db-backup:latest

Docker:
- Docker version 24.0.2, build cb74dfc
- Docker Compose version v2.18.1

OS: macOS 13.4

**Comment by ToshY** [comment]
The `EXTRA_OPTS` is no longer used to check the database availability (see [PR 209](https://github.com/tiredofit/docker-db-backup/pull/209/files)), so that can not be the root cause of your issue. However, your error log shows that `gse-db` is not accesible, so I would reckon it's more likely that your database container is not in the same network as your `db-backup` container. If you post your docker compose config it would be easier to debug the problem.

**Comment by dabico** [comment]
Yeah, I'm not sure what the cause of the original issue was. At that moment I remember switching to a `mysql.cnf` configuration in which I specified all the parameters for `mysqldump`. Since then I have not been having this issue. I tried to make a [reproducible example](https://github.com/dabico/docker-db-backup-issue-234), but it seems that even there I can not recreate the original issue that I had. In any case, I will close this since it is no longer valid.
